apps/src/StudioApp.js

```diff
@@ -2759,7 +2760,8 @@ StudioApp.prototype.setPageConstants = function (config, appSpecificConstants) {
     config.level.markdownInstructions,
     config.locale,
     !!config.noInstructionsWhenCollapsed,
-    !!config.showInstructionsInTopPane
+    !!config.showInstructionsInTopPane,
+    !!config.level.inputOutputTable
```

Comment:
FWIW, this will probably change in my next PR anyways, as I'm adding yet another param, and I think it starts to make more sense to just pass in the config option than all these params.